stdlib/public/SDK/Foundation/AffineTransform.swift

```diff
@@ -124,7 +124,7 @@ public struct AffineTransform : ReferenceConvertible, Hashable, CustomStringConv
          [    0       0    1 ]
      */
     public init(rotationByDegrees angle: CGFloat) {
-        let α = Double(angle) * M_PI / 180.0
+        let α = Double(angle) * .pi / 180.0
         self.init(rotationByRadians: CGFloat(α))
```

Comment:
These could be cleaned up further, I think; there's no long any need for the conversions.

    self.init(rotationByRadians: angle * .pi / 180)

Possibly outside the scope of this change.